docs/tutorials/data-cli.md

```diff
@@ -0,0 +1,42 @@
+# Data Module (CLI)
```

Comment:
is this supposed to show up in the tutorials yet in vuepress? im assuming since the readme is blank, this is for later?

Reply:
It is not meant to be included, the data tutorials were written back in January and need to be updated. Thanks for reviewing though! I was just moving them to their own files to be updated later, see https://github.com/regen-network/regen-ledger/pull/453#discussion_r682960870.